### ISSUE TITLE
bug: Latest version does not display table UI correctly when using SQL

### ISSUE THREAD

**Issue by juan-carvajal** [parseablehq/parseable#676]
Version info:
![image](https://github.com/parseablehq/parseable/assets/39658930/1d08486b-02cf-4a80-a06f-f19781c7b475)
Query: `SELECT left(msg,100) as msg, time, trace_id, error,details,status FROM logstashlogs order by time desc, trace_id LIMIT 9000 OFFSET 0`
Response (truncated): 
```
[
    {
        "msg": "request to http://opentelemetry-collector.lightops-obs-agent:9411/api/v2/spans failed: Post \"http://",
        "time": "2024-02-27T02:55:52.170620275Z"
    },
    {
        "msg": "about to send a POST request to http://opentelemetry-collector.lightops-obs-agent:9411/api/v2/spans ",
        "time": "2024-02-27T02:55:52.042079674Z"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:49.701818801Z",
        "trace_id": "790845248146244649e7f41d7ecbe11c",
        "status": 200
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:49.504646071Z",
        "trace_id": "48d804448a6b18180c6072de58d36388",
        "status": 200
    },
    {
        "msg": "entitlementsGroupsArgs",
        "time": "2024-02-27T02:55:49.351643406Z",
        "trace_id": "790845248146244649e7f41d7ecbe11c"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:49.226356318Z",
        "status": 202
    },
    {
        "msg": "entitlementsGroupsArgs",
        "time": "2024-02-27T02:55:49.10655436Z",
        "trace_id": "48d804448a6b18180c6072de58d36388"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.977103979Z",
        "trace_id": "ee3203cd2146a09643336d4922c9d11b",
        "status": 202
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.96543672Z",
        "trace_id": "1eea676a11d338bc4cf3f37097148637",
        "status": 200
    },
    {
        "msg": "entitlementsGroupsArgs",
        "time": "2024-02-27T02:55:48.893409242Z"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.816802965Z",
        "status": 200
    },
    {
        "msg": "missing or malformed auth principal",
        "time": "2024-02-27T02:55:48.816518481Z",
        "details": "no Authorization header found"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.814835888Z",
        "trace_id": "2b40ac6bf8c6d55dab9183e6a9163921",
        "status": 200
    },
    {
        "msg": "entitlementsGroupsArgs",
        "time": "2024-02-27T02:55:48.777004617Z",
        "trace_id": "ee3203cd2146a09643336d4922c9d11b"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.764379309Z",
        "status": 400
    },
    {
        "msg": "failed to parse request",
        "time": "2024-02-27T02:55:48.764073234Z",
        "error": "Framework: dummy does not validate as in(python37|model)"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.648479284Z",
        "trace_id": "8e6095ee69d4d76c9eae66f7935fe2f5",
        "status": 200
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.641132667Z",
        "trace_id": "dd154b30730579c9ccbb55c5f31f049b",
        "status": 400
    },
    {
        "msg": "failed to parse request",
        "time": "2024-02-27T02:55:48.640876896Z",
        "trace_id": "dd154b30730579c9ccbb55c5f31f049b",
        "error": "DataPartitionId: non zero value required"
    },
    {
        "msg": "entitlementsGroupsArgs",
        "time": "2024-02-27T02:55:48.613545349Z",
        "trace_id": "1eea676a11d338bc4cf3f37097148637"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.512402018Z",
        "trace_id": "60c3978ff01d22619a06a3d59a11b85c",
        "status": 400
    },
    {
        "msg": "failed to parse request",
        "time": "2024-02-27T02:55:48.512268417Z",
        "trace_id": "60c3978ff01d22619a06a3d59a11b85c",
        "error": "Submitter: non zero value required"
    },
    {
        "msg": "Unable to parse token",
        "time": "2024-02-27T02:55:48.512162447Z",
        "trace_id": "60c3978ff01d22619a06a3d59a11b85c"
    },
    {
        "msg": "missing or malformed auth principal",
        "time": "2024-02-27T02:55:48.511869276Z",
        "trace_id": "60c3978ff01d22619a06a3d59a11b85c",
        "details": "invalid token format"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.509733203Z",
        "trace_id": "922b8eaccc92b707bc93d61826690b31",
        "status": 404
    },
    {
        "msg": "failed to download model",
        "time": "2024-02-27T02:55:48.509609841Z",
        "trace_id": "922b8eaccc92b707bc93d61826690b31",
        "error": "failed to find model of framework=python37 with id=SKLearnServernotfound and version=1709002547881501141."
    },
    {
        "msg": "could not download model",
        "time": "2024-02-27T02:55:48.509429432Z",
        "trace_id": "922b8eaccc92b707bc93d61826690b31",
        "error": "mongo: no documents in result"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.482809632Z",
        "trace_id": "5e649cdf4d860199dfd8edb2cca95edb",
        "status": 400
    },
    {
        "msg": "failed to parse request",
        "time": "2024-02-27T02:55:48.482683845Z",
        "trace_id": "5e649cdf4d860199dfd8edb2cca95edb",
        "error": "Submitter: non zero value required"
    },
    {
        "msg": "Unable to parse token",
        "time": "2024-02-27T02:55:48.482567086Z",
        "trace_id": "5e649cdf4d860199dfd8edb2cca95edb"
    },
    {
        "msg": "missing or malformed auth principal",
        "time": "2024-02-27T02:55:48.482258345Z",
        "trace_id": "5e649cdf4d860199dfd8edb2cca95edb",
        "details": "invalid token format"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.47994614Z",
        "trace_id": "84a0f2e11ac46e8ca935e4d40188048f",
        "status": 400
    },
    {
        "msg": "failed to parse request",
        "time": "2024-02-27T02:55:48.479736576Z",
        "trace_id": "84a0f2e11ac46e8ca935e4d40188048f",
        "error": "DataPartitionId: non zero value required"
    },
    {
        "msg": "entitlementsGroupsArgs",
        "time": "2024-02-27T02:55:48.469227536Z",
        "trace_id": "2b40ac6bf8c6d55dab9183e6a9163921"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.383039993Z",
        "trace_id": "92dbf42ebe85526d76d4a4a0e3cd0aba",
        "status": 200
    },
    {
        "msg": "response payload too large, compressing",
        "time": "2024-02-27T02:55:48.381022842Z"
    },
    {
        "msg": "Downloading folder",
        "time": "2024-02-27T02:55:48.377938946Z",
        "trace_id": "92dbf42ebe85526d76d4a4a0e3cd0aba"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.351966343Z",
        "trace_id": "e8d05900229fa8c78e4c4bbc189fac9d",
        "status": 404
    },
    {
        "msg": "failed to get model group",
        "time": "2024-02-27T02:55:48.351752661Z",
        "trace_id": "e8d05900229fa8c78e4c4bbc189fac9d",
        "error": "failed to find model group in mongo with id=e2e2b60d51709002547mgnotfound."
    },
    {
        "msg": "get models. Invalid datapartition",
        "time": "2024-02-27T02:55:48.351506528Z",
        "trace_id": "e8d05900229fa8c78e4c4bbc189fac9d"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.339645917Z",
        "trace_id": "20559cd189037287c033ae64de32ee6e",
        "status": 400
    },
    {
        "msg": "failed to parse request",
        "time": "2024-02-27T02:55:48.339393022Z",
        "trace_id": "20559cd189037287c033ae64de32ee6e",
        "error": "DataPartitionId: non zero value required"
    },
    {
        "msg": "entitlementsGroupsArgs",
        "time": "2024-02-27T02:55:48.301176787Z",
        "trace_id": "8e6095ee69d4d76c9eae66f7935fe2f5"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.226418896Z",
        "trace_id": "0ae51c6f0fb00f95b9a26a813eba2801",
        "status": 200
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.212111847Z",
        "trace_id": "72931663cd3dc9e897e055d64bb3858b",
        "status": 400
    },
    {
        "msg": "failed to parse request",
        "time": "2024-02-27T02:55:48.21194843Z",
        "trace_id": "72931663cd3dc9e897e055d64bb3858b",
        "error": "Submitter: non zero value required"
    },
    {
        "msg": "Unable to parse token",
        "time": "2024-02-27T02:55:48.211830148Z",
        "trace_id": "72931663cd3dc9e897e055d64bb3858b"
    },
    {
        "msg": "missing or malformed auth principal",
        "time": "2024-02-27T02:55:48.211710172Z",
        "trace_id": "72931663cd3dc9e897e055d64bb3858b",
        "details": "invalid token format"
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.210389049Z",
        "trace_id": "daff17673f71d3c46717a9c47815d428",
        "status": 200
    },
    {
        "msg": "request log",
        "time": "2024-02-27T02:55:48.175153496Z",
        "trace_id": "60d8135eb37501df0afb7f3eb9630727",
        "status": 201
    },
    {
        "msg": "Writing file",
        "time": "2024-02-27T02:55:48.171062208Z",
        "trace_id": "60d8135eb37501df0afb7f3eb9630727"
    },
    {
        "msg": "Writing file",
        "time": "2024-02-27T02:55:48.170838056Z",
        "trace_id": "60d8135eb37501df0afb7f3eb9630727"
    },
    {
        "msg": "Writing file",
        "time": "2024-02-27T02:55:48.170620828Z",
        "trace_id": "60d8135eb37501df0afb7f3eb9630727"
    },
    {
        "msg": "Writing file",
        "time": "2024-02-27T02:55:48.170206069Z",
        "trace_id": "60d8135eb37501df0afb7f3eb9630727"
    },
    {
        "msg": "Writing file",
        "time": "2024-02-27T02:55:48.170180049Z",
        "trace_id": "60d8135eb37501df0afb7f3eb9630727"
    }
]
```
UI:
![image](https://github.com/parseablehq/parseable/assets/39658930/01fe6277-7fce-4c66-9a14-298cf4d5b025)

Expected behavior: Table shows all request columns in the query, even if some are empty
Behavior: Table is only showing columns where ALL row have values. Not very useful when looking for problems or outliers in the logs

**Comment by nitisht** [comment]
Thanks for reporting @juan-carvajal . We'll take a look, cc @balaji-jr

**Comment by balaji-jr** [comment]
@juan-carvajal 
Thanks for reporting. This issue will be fixed in the upcoming release. 
 refer: [aa3990f](https://github.com/parseablehq/console/pull/204/commits/aa3990f4bc2b5492edfe461c705d275611bd9ee5) & [efbd2b7](https://github.com/parseablehq/console/pull/204/commits/efbd2b7d7a361955b874a2fb06468febc0e1d248)

**Comment by nitisht** [comment]
Fixed now, will be available in v0.9.0
